api_tests/src/payload.ts

```diff
@@ -1,19 +1,58 @@
-/**
- * cnd's REST API swap response for the split protocols (han, herc20, halight).
- *
- * Defined here until it is stabilized. Once stabilized, it can be moved to the SDK.
- */
-
 import {
     Action,
     EmbeddedRepresentationSubEntity,
     Entity,
     Link,
 } from "comit-sdk/dist/src/cnd/siren";
+import { Peer } from "comit-sdk";
+
+/**
+ * Payloads for the `/swaps/` REST API.
+ */
+
+/*
+ * The payload for POST requests to create a swap on the cnd REST API.
+ */
+interface Create<A, B> {
```

Comment:
I preferred the name `Payload` to be honest. `Create` is a verb, which makes it awkward as an instance of data, no?

Reply:
mmm, good point on being a verb. It was `RequestBody` before, was looking for a shorter name but still something that says its the payload of a POST request that creates a swap. Used `Payload` as suggested.